src/frontend/token.cpp

```diff
@@ -34,6 +34,8 @@ namespace graphit {
                 return "'-'";
             case Token::Type::INT:
                 return "'int'";
+            case Token::Type::UINT_64:
```

Comment:
Can you also add a case here for UINT? This is required for dumping the tokens as string.